R/grid_code_paths.R

```diff
@@ -24,45 +24,49 @@ tune_grid_loop <- function(resamples, grid, workflow, metrics, control, rng) {
   }
 
   if (identical(parallel_over, "resamples")) {
-    results <- foreach::foreach(
-      iteration = iterations,
-      .packages = packages,
-      .errorhandling = "pass"
-    ) %op% {
-      tune_grid_loop_iter_safely(
-        iteration = iteration,
-        resamples = resamples,
-        grid_info = grid_info,
-        workflow = workflow,
-        metrics = metrics,
-        control = control,
-        seeds = seeds
-      )
-    }
-  } else if (identical(parallel_over, "everything")) {
-    results <- foreach::foreach(
-      iteration = iterations,
-      .packages = packages,
-      .errorhandling = "pass"
-    ) %:%
-      foreach::foreach(
-        row = rows,
+    suppressPackageStartupMessages(
```

Comment:
I don't see why you need `suppressPackageStartupMessages()` here. Shouldn't it _only_ be used with `load_pkgs()` and `load_namespace()` in `tune_grid_loop_iter()`?

Reply:
We still need it since we use `foreach(.packages = packages)`. That doesn't use the loading functions in `tune`

Reply:
😭